roles/etcd_client_certificates/meta/main.yml

```diff
@@ -14,3 +14,4 @@ galaxy_info:
   - system
 dependencies:
 - role: etcd_common
+- role: delegated_serial_command
```

Comment:
I'll work those in a separate PR. What's the verdict on:

* Multiple modules housed in a single role.
* Prefixing the name of this role as module only in some way if it exists only for `delegated_serial_command`.